### ISSUE TITLE
Request custom logo for our DAO

### ISSUE THREAD

**Issue by lkngtn** [1Hive/deployment-log#6]
A handful of Aragon organizations already have custom logos (eg https://mainnet.aragon.org/#/governance.aragonproject.eth/0x277bfcf7c2e162cb1ac3e9ae228a3132a75f83d4) 

AFAIK these are manually added somewhere in the aragon client codebase (though it is planned to make this configurable by orgs themselves), when we are on mainnet we should request a custom logo if the feature hasn't be broadly enabled yet.

**Comment by AaronAbuUsama** [comment]
This can be closed now @burrrata

**Comment by lkngtn** [comment]
I don't think it can be. We don't have a mainnet dao and will need to repeat the process afaik
